src/class-wpml-gutenberg-strings-registration.php

```diff
@@ -76,6 +90,8 @@ private function register_blocks( array $blocks, array $package_data ) {
 					$string->type
 				);
 
+				$string_id = apply_filters( 'wpml_string_id_from_package', 0, $package_data, $string->id, $string->value );
```

Comment:
@brucepearson, I think you can move this line to just before 106.

Reply:
@strategio Done